src/topology/algebra/module/basic.lean

```diff
@@ -238,6 +242,22 @@ instance {M' : Type*} [add_comm_monoid M'] [module R M'] [uniform_space M']
   complete_space U.topological_closure :=
 is_closed_closure.complete_space_coe
 
+lemma submodule.is_closed_or_dense_of_is_coatom (s : submodule R M) (hs : is_coatom s) :
```

Comment:
this one would deserve a docstring, because the terminology `is_coatom` will probably not be standard for readers/users of this file.